demo-app/src/components/DataAndFriends/ToolbarRegion.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from 'grommet';
+
+export const ToolbarRegion = ({ gap, ...rest }) => {
+  return <Box direction="row" gap={gap} {...rest} />;
+};
+
+ToolbarRegion.propTypes = {
+  gap: PropTypes.string,
+};
+
+ToolbarRegion.defaultProps = {
```

Comment:
React is going to be deprecating defaultProps. Should we use do `gap = "medium"` in the destructuring above instead?

Reply:
Thanks for the reminder.

Reply:
done